src/containers/Nav/Nav.js

```diff
@@ -4,10 +4,14 @@ import './Nav.css'
 
 const Nav = () => {
     return(
-        <nav>
-            Testing Nav
+        <nav className="nav">
+            <p className="nav-p">FAVORITES</p>
```

Comment:
Not sure if p tags are as SEO friendly so to speak for a nav, I will talk to Tyler - SEO specialist - and see what feedback I get. Also from an accessibility point as well, unsure how friendly this is for screen readers.